src/pages/transactions/TransactionForm.js

```diff
@@ -51,9 +52,13 @@ export default function TransactionForm({ onAddTransaction, onAlert }) {
     <div>
       <h1>Add Transaction</h1>
       <form onSubmit={handleSubmit}>
-        <button type="button" onClick={() => router.push("/")}>
+        <Button
+          variant="secondary"
+          type="button"
+          onClick={() => router.push("/")}
+        >
           Cancel
-        </button>
+        </Button>
```

Comment:
QA issues: If you add a new entry and then click on it and reload your browser, it says: "Transaction not found" instead of showing the entry.